packages/ic-management/candid/ic-management.did

```diff
@@ -23,6 +24,7 @@ type definite_canister_settings = record {
     freezing_threshold : nat;
     reserved_cycles_limit : nat;
     log_visibility : log_visibility;
+    wasm_memory_limit : nat;
```

Comment:
This is also a breaking change.
I'm asking the author of https://github.com/dfinity/interface-spec/pull/278 if this was intentional.

Reply:
I'm told this is not breaking because this is only used as an output type.
But if the client updates their candid before the backend does, then the client expects the new field while it's not there so it is breaking.

Reply:
I'm told it's rolled out already.